target/scripts/helpers/accounts.sh

```diff
@@ -135,7 +135,7 @@ function _create_dovecot_alias_dummy_accounts() {
       fi
 
       DOVECOT_USERDB_LINE="${ALIAS}:${REAL_ACC[1]}:${DMS_VMAIL_UID}:${DMS_VMAIL_GID}::/var/mail/${REAL_DOMAINNAME}/${REAL_USERNAME}/home::${REAL_ACC[2]:-}"
-      if grep -qi "^${ALIAS}:" "${DOVECOT_USERDB_FILE}"; then
+      if grep -qxF "${DOVECOT_USERDB_LINE}" "${DOVECOT_USERDB_FILE}"; then
```

Comment:
I don't think there's any harm in using `-i` still. We don't want to introduce aliases that differ by casing as that tends to introduce bugs.

```suggestion
      # Match a full line with `-xF` to avoid regex patterns introducing false positives matching `ALIAS`:
      if grep -qixF "${DOVECOT_USERDB_LINE}" "${DOVECOT_USERDB_FILE}"; then
```